tests/test_jupyter_notebook/env.yaml

```diff
@@ -4,4 +4,5 @@ channels:
 dependencies:
   - python >=3.5
   - jupyter
+  - papermill
   - nbformat <5.5 # there is a bug in 5.5 that leads to a crash during notebook validation
```

Comment:
Is nbformat still needed in the env?

Reply:
I assume so, as we use the `nbformat` Python package throughout `notebook.py`. But it's possible that the version restriction here is no longer needed.

Reply:
Huh, looks like this is already a dependency of jupyter and may have been only included to deal with the aforementioned bug. Removed this from all environment YAMLs and the tests still pass.